portfolio/src/main/webapp/projects_style.css

```diff
@@ -0,0 +1,32 @@
+/* all clickable cards on the projects page */
+.card {
+  background-color: #ececec;
+  display: flex;
+  height: 150px;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+
+h3 {
+  margin: 0px 5px;
```

Comment:
No "px" is required with 0. This should be:
`margin: 0 5px;`
Here and throughout the file.

Reply:
Thank you, fixed.